checks/webhook.go

```diff
@@ -0,0 +1,66 @@
+// Copyright 2022 Security Scorecard Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package checks
+
+import (
+	"fmt"
+
+	"github.com/ossf/scorecard/v4/checker"
+	sce "github.com/ossf/scorecard/v4/errors"
+)
+
+const (
+	// CheckWebHooks is the registered name for WebHooks.
+	CheckWebHooks = "Webhooks"
+)
+
+//nolint:gochecknoinits
+func init() {
+	if err := registerCheck(CheckWebHooks, WebHooks, nil); err != nil {
+		// this should never happen
+		panic(err)
+	}
+}
+
+// WebHooks run Contributors check.
+func WebHooks(c *checker.CheckRequest) checker.CheckResult {
+	hooks, err := c.RepoClient.ListWebhooks()
+	if err != nil {
+		e := sce.WithMessage(sce.ErrScorecardInternal, fmt.Sprintf("Client.Repositories.ListWebhooks: %v", err))
+		return checker.CreateRuntimeErrorResult(CheckWebHooks, e)
+	}
+
+	if len(hooks) < 1 {
+		return checker.CreateMaxScoreResult(CheckWebHooks, "no webhooks defined")
+	}
+
+	hasSecretToCount := 0
```

Comment:
Nit: this variable counts "number of webhooks with *no* secrets". Consider renaming to reflect that - something like `hasNoSecretCount`. Right now the var name and the value it holds are different.